### DISCUSSION TITLE
Various apps statistics  visible on a single app dashboard for Administrator

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10562] [Help] [answered]
Hi

i have 2 user apps.

App A
App B

rows are added and deleted in both Apps. 
with time rows will be in thousands

i want to see how many rows were added and deleted in each app daily/monthly/Grandtotal for each App.

Can these statistic be seen in a separate app by admin (like administrator dashboard for all user apps)?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @sghriyadh,

This is something that should be possible with audit logs. You can do something similar at the moment with the created rows. However, I have created a enhancement to add the ability to do this with deleted rows as well. This can be found here: https://github.com/Budibase/budibase/issues/10567 

Using this would allow you to find all deleted rows on a day/month or year depending on your search parameters. (Screenshot below shows the different parameters available for this.)

You can also take this a step further and download a CSV containing the related audit logs.

![Screenshot 2023-05-12 at 15 54 37](https://github.com/Budibase/budibase/assets/126772285/88d932a5-f611-42a9-a6a5-27ec8e58251e)

As for the security aspect of your question, these logs are only viewable by users with the roles admin or developer.

For any additional information on this you can read here: https://docs.budibase.com/docs/audit-logs

Hopefully this helps!
